test/tracking/test_planar_tracking.py

```diff
@@ -38,11 +38,15 @@ def test_real(self, device, dtype, data):
             if isinstance(data[k], torch.Tensor):
                 data[k] = data[k].to(device, dtype)
         h0, w0 = data["image0"].shape[2:]
-        data["image0"] = resize(data["image0"], (int(h0 // 2), int(w0 // 2)))
-        data["image1"] = resize(data["image1"], (int(h0 // 2), int(w0 // 2)))
+        data["image0"] = resize(
+            data["image0"], (int(h0 // 2), int(w0 // 2)), interpolation='bilinear', align_corners=False
+        )
+        data["image1"] = resize(
+            data["image1"], (int(h0 // 2), int(w0 // 2)), interpolation='bilinear', align_corners=False
+        )
         with torch.no_grad():
             tracker.set_target(data["image0"])
-            torch.random.manual_seed(0)
+            torch.manual_seed(3)  # issue kornia#2027
```

Comment:
maybe, here there must be a xfail -- the crash occurs related to ransac, I don't know if the seed will be enough to avoid crashes